caom2/caom2/plane.py

```diff
@@ -852,7 +894,7 @@ def __init__(self, name,
         self.last_executed = last_executed
 
         self._keywords = set()
-        self._inputs = caom_util.TypedSet(PlaneURI, )
+        self._inputs = caom_util.TypedSet(str, )
```

Comment:
`TypedSet` seems like overkill for `str`

Reply:
To make sure `PlaneURI`s or `Plane`s are not passed in by mistake.